src/preamble.js

```diff
@@ -930,11 +930,11 @@ function stackTrace() {
 
 // Memory management
 
-var PAGE_SIZE = 4096;
+var PAGE_SIZE = 65536; // Wasm pages are fixed to 64k.
```

Comment:
>But for wasm it will be impractical to actually implement 4k granularity on top of 16k underlying pages. So given that I think it would be best to go to 16k everywhere.

Wasm heap sizes are multiples of 64k, that is what I followed here. Though I don't mind much what the value of `PAGE_SIZE` here is, I just made it to match the heap size granule. Certainly ok to detach that if desired, although if we report 16k pages, we can't actually grow just by one page. Is that ok?